### PR TITLE
Don't add loopback ip2me route again if already configured

### PR DESCRIPTION
**Pull request by jipanyang**
Signed-off-by: Jipan Yang <jipan.yang@alibaba-inc.com>

**What I did**
Add check to see if lo interface already processed. If yes, skip addIp2MeRoute() for it.

**Why I did it**
It is possible lo interface request comes again,  or comes after warm restore in case of warm reboot.

**How I verified it**

**Details if related**

### CODE REVIEW

File: orchagent/intfsorch.cpp [hunk 1]
```diff
@@ -174,7 +174,16 @@ void IntfsOrch::doTask(Consumer &consumer)
         {
             if (alias == "lo")
             {
-                addIp2MeRoute(vrf_id, ip_prefix);
+                // set request for lo may come after warm start restore
+                auto it_intfs = m_syncdIntfses.find(alias);
+                if (it_intfs == m_syncdIntfses.end())
```
[the remark this answers is not in the document]
Comment (jipanyang) [reply orchagent/intfsorch.cpp]: Correct.   Will provide more update for the change to address the points you brought up.

File: orchagent/intfsorch.cpp [hunk 2]
```diff
@@ -261,7 +292,15 @@ void IntfsOrch::doTask(Consumer &consumer)
         {
             if (alias == "lo")
             {
-                removeIp2MeRoute(vrf_id, ip_prefix);
+                if (m_syncdIntfses[alias].ip_addresses.count(ip_prefix))
```
Comment (nikos-github) [inline orchagent/intfsorch.cpp]: In the set operation, you skip adding the loopback in m_syncdIntfses if the key (loopback) has no prefix. I think you will crash here.

Comment (jipanyang) [reply orchagent/intfsorch.cpp]: right.  There is also vrf_id issue to be taken care of.

File: orchagent/intfsorch.cpp [hunk 3]
```diff
@@ -174,7 +174,38 @@ void IntfsOrch::doTask(Consumer &consumer)
         {
             if (alias == "lo")
             {
-                addIp2MeRoute(vrf_id, ip_prefix);
+                if (!ip_prefix_in_key)
```
[the remark this answers is not in the document]
Comment (jipanyang) [reply orchagent/intfsorch.cpp]: The ip addresses on loopback come from configuration.  At least for current warm reboot implementation, we don't expect any configuration change during the warm reboot window (util restore is finished). 

Other than configuration change, in what scenario the IP for loopback will gone?

[the remark this answers is not in the document]
Comment (nikos-github) [reply orchagent/intfsorch.cpp]: If we are not disallowing the config changes, then all IPs should be removed in this case. Warm reboot can also result in this situation with config change, correct?

**Pull request merged**